### ISSUE TITLE
小程序等调用系统摄像头无法替换

### ISSUE THREAD

**Issue by LayFz** [w2016561536/android_virtual_cam#181]
在使用微信小程序以及qq小程序中，需要调用系统相机拍照，此时无法替换系统相机。

**Comment by reptilano** [comment]
Have you checked this video? www.youtube.com/watch?v=11fz-ssXqWQ&ab_channel=Swapcam

**Comment by LayFz** [comment]
Yes, I watched the video, but the video only showed the effect of the program. Currently, I am trying to use Tencent’s QQ applet to make the camera call unable to be replaced.

**Comment by apex-cmyk** [comment]
你好，你可以尝试用云手机登录微信和QQ以后进行尝试呢。云手机平台可以对调用系统摄像头还是程序连接的摄像头进行配置的。目前这个功能上，亚矩阵云手机平台是做的比较不错的，相比较华为云手机、百度云手机更实用易用和稳定一些呢。
